snake_env.py

```diff
@@ -22,6 +42,7 @@ def __init__(self, render_mode=None, seed=None, size=8):
         # Target's location, Neo's location and length
         self.observation_space = spaces.Dict({
```

Comment:
This needs to correspond with the changes made to obs space

Reply:
Not sufficient, still missing `grid`

Reply:
Done by Norbi!